### PR TITLE
More layer indicators in the layers panel

### PR DESCRIPTION
**Pull request by DelazJ**
Goal:

Ticket(s): #

- [ ] Backport to LTR documentation is required

### CODE REVIEW

File: docs/user_manual/introduction/general_tools.rst [hunk 1]
```diff
@@ -301,6 +303,7 @@ symbols are:
   close this project. To avoid data loss and make the layer permanent, click
   the icon to store the layer in any of the OGR vector formats supported by QGIS.
 * |indicatorNoCRS| to identify a layer that has no/unknown CRS
+* |indicatorTemporal| to identify a temporal layer controlled by canvas animation
```
Comment (havatv) [inline docs/user_manual/introduction/general_tools.rst:306]:
```suggestion
* |indicatorTemporal| to identify a temporal layer
```

Comment (DelazJ) [reply docs/user_manual/introduction/general_tools.rst:306]: why?

Comment (havatv) [reply docs/user_manual/introduction/general_tools.rst:306]: I assumed that it was used to identify a temporal layer, not depending on the context (or how QGIS is handling it).

Comment (DelazJ) [reply docs/user_manual/introduction/general_tools.rst:306]: https://github.com/qgis/QGIS/commit/57988a4dc1dca985eaca40a6fd21196467f11b4f

**Pull request merged**